src/Command/DumpCommand.php

```diff
@@ -0,0 +1,121 @@
+        ;
+
+        parent::configure();
+    }
+
+    protected function runCommand(InputInterface $input, OutputInterface $output)
+    {
+        $ignoredTables = array_map('trim', explode(',', $input->getOption('ignore-tables') . ',' . $this->config->getLogTableName() ? : $this->config->getLogTableName()));
+        $output->writeln('');
+
+        $indent = $this->getIndent($input);
+
+        $migration = '';
+        $tables = $this->getOrderedTables($ignoredTables);
+        foreach ($tables as $table) {
+            $migration .= $indent . "\$this->table('{$table->getName()}')\n";
+            foreach ($table->getColumns() as $column) {
+                $migration .= "$indent$indent" . "->addColumn('{$column->getName()}', '{$column->getType()}')\n";
+            }
+            foreach ($table->getIndexes() as $index) {
+                $migration .= "$indent$indent" . "->addIndex(";
+                $indexColumns = $index->getColumns();
+                $migration .= $this->columnsToString($indexColumns) . ", '" . strtolower($index->getType()) . "', '" . strtolower($index->getMethod()) . "', '{$index->getName()}')\n";
+            }
+            foreach ($table->getForeignKeys() as $foreignKey) {
+                $migration .= "$indent$indent" . "->addForeignKey(";
+                $migration .= $this->columnsToString($foreignKey->getColumns()) . ", '{$foreignKey->getReferencedTable()}', ";
+                $migration .= $this->columnsToString($foreignKey->getReferencedColumns()) . ", '{$foreignKey->getOnDelete()}', '{$foreignKey->getOnUpdate()}')\n";
+            }
+            $migration .= "$indent$indent" . "->create();\n\n";
+        }
+
+        if ($input->getOption('data')) {
+            $ignoredDataTables = $input->getOption('ignore-data-tables')
+                ? array_merge($ignoredTables, array_map('trim', explode(',', $input->getOption('ignore-data-tables'))))
+                : $ignoredTables;
+
+            foreach ($tables as $table) {
+                if (in_array($table->getName(), $ignoredDataTables)) {
+                    continue;
+                }
+                $rows = $this->adapter->fetchAll($table->getName());
+                if (empty($rows)) {
+                    continue;
+                }
+                $migration .= "$indent\$this->insert('{$table->getName()}', [\n";
+                foreach ($rows as $row) {
+                    $migration .= "$indent$indent" . "[\n";
+                    foreach ($row as $column => $value) {
+                        $migration .= "$indent$indent$indent'$column' => '" . addslashes($value) . "',\n";
+                    }
+                    $migration .= "$indent$indent],\n";
+                }
+                $migration .= "$indent]);\n\n";
+            }
+        }
+
+        $output->write($migration);
+
+        $output->writeln('');
+    }
+
+    private function getOrderedTables(array $ignoredTables = [])
```

Comment:
Shouldn't  this be named `getFilteredTables()` rather than ordered? Currently it uses a blacklist *filter*ing against `$ignoredTables`